gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -123,15 +128,31 @@ def get_sub_sheet_titles_to_ids(sheet_api_instance):
     global subsheet_titles_to_ids
     if subsheet_titles_to_ids:
         return subsheet_titles_to_ids
-    print("Retrieving subsheet titles to ids")
-    sheets = sheet_api_instance.get(spreadsheetId=SPREADSHEET_ID, fields='sheets/properties').execute()
+    logger.info("Retrieving subsheet titles to ids")
+    request = sheet_api_instance.get(spreadsheetId=SPREADSHEET_ID, fields='sheets/properties')
+    sheets = request.execute()
     subsheet_titles_to_ids = {sheet['properties']['title']: sheet['properties']['sheetId'] for sheet in
                                sheets['sheets']}
     return subsheet_titles_to_ids
 
+def is_429_error(exception):
+    return isinstance(exception, HttpError) and exception.resp.status == 429
```

Comment:
Can you also log what exception is actually being thrown? If we see other errors being thrown during production, we need to investigate why those errors happen.

Reply:
Errors other than 429 errors are being logged at [this](https://github.com/AFA-Tooling/GradeSync/blob/318029dbdd6a3afa17490a85cc87a00a0417857b/gradescopeCronJob/gradescope_to_spreadsheet.py#L119) line

Reply:
I added suggestions